.github/workflows/dependency-scan.yml

```diff
@@ -20,8 +20,9 @@ jobs:
           cabal update
 
       - name: Install spectrometer from github
+        # To get this to work, we have to run the local script and then change it after.
         run: |
-          curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spectrometer/master/install.sh | bash
+          ./install.sh -d
```

Comment:
Is this still required? Shouldn't the script change fix the installation issue?

Reply:
No, it has to be merged first, it's picking up the script from master, which hasn't changed and is still seeing the issue.

Reply:
Gotcha, I just wanted to make sure that it wouldn't be an issue for others still.